hapi-proto/src/main/proto/ScheduleSign.proto

```diff
@@ -27,6 +27,13 @@ option java_multiple_files = true;
 
 import "BasicTypes.proto";
 
+/*
+Provides signatures for a given Scheduled Transaction. The Scheduled Transaction executes if all of the required signatures are witnessed.
+
+INVALID_SCHEDULE_ID is returned if the specified ScheduleID does not exist, already executed or expired
```

Comment:
If some key change has been made in the to-be-executed transaction, this can result in 
* `INVALID_ACCOUNT_ID `, if there is an explicit payer
* `UNRESOLVABLE_REQUIRED_SIGNERS`, if any of the "other" parties' keys have resulted in an error.